File: result/translate/chatgpt/java2python/python/FIRST_ELEMENT_OCCURRING_K_TIMES_ARRAY.py
def f_gold(arr, n, k):
    count_map = {}
    
    for i in range(n):
        a = 0
        if arr[i] in count_map:
            a = count_map[arr[i]]
        count_map[arr[i]] = a + 1
        
    for i in range(n):
        if count_map[arr[i]] == k:
            return arr[i]
    
    return -1

File: result/translate/transcoder/python2java/java/FIRST_ELEMENT_OCCURRING_K_TIMES_ARRAY.java
public static int f_gold ( int [ ] arr , int n , int k ) {
  HashMap < Integer , Integer > countMap = new HashMap < Integer , Integer > ( ) ;
  ;
  for ( int i = 0 ;
  i <= n ;
  i ++ ) {
    if ( ( arr [ i ] == 0 ) || ( arr [ i ] == 1 ) ) {
      countMap . put ( arr [ i ] , 1 ) ;
    }
    else {
      countMap . put ( arr [ i ] , 1 ) ;
    }
    i ++ ;
  }
  for ( int i = 0 ;
  i <= n ;
  i ++ ) {
    if ( ( countMap . get ( arr [ i ] ) == k ) || ( countMap . get ( arr [ i ] ) == 1 ) ) {
      return arr [ i ] ;
    }
    i ++ ;
  }
  return - 1 ;
}
